tests/disease/test_disease.py

```diff
@@ -554,3 +557,99 @@ def test_artifact_transition_keys(mocker, disease):
     # check remission rate
     remissive_transition = with_condition.add_rate_transition(healthy)
     assert remissive_transition._rate_source == f"cause.{cause}.remission_rate"
+
+
+@pytest.mark.parametrize("rate_conversion_type", ["linear", "exponential"])
+def test_transition_rate_to_probability_configuration(
+    base_config: LayeredConfigTree,
+    base_plugins: LayeredConfigTree,
+    disease: str,
+    rate_conversion_type: str,
+):
+    """
+    Test that the transition rate to probability configuration is set correctly.
+    """
+    healthy = BaseDiseaseState("healthy")
+    sick = DiseaseState("sick")
+    key = "sequela.acute_myocardial_infarction_first_2_days.incidence_rate"
+    transition = RateTransition(
+        input_state=healthy,
+        output_state=sick,
+        get_data_functions={"incidence_rate": lambda builder, _: builder.data.load(key)},
+    )
+    healthy.transition_set.append(transition)
+    model = DiseaseModel(disease, initial_state=healthy, states=[healthy, sick])
+
+    base_config.update(
+        {
+            f"{transition.name}": {
+                "rate_conversion_type": rate_conversion_type,
+            }
+        }
+    )
+
+    # Sets the configuration
+    sim = InteractiveContext(
+        components=[TestPopulation(), model],
+        configuration=base_config,
+        plugin_configuration=base_plugins,
+        setup=True,
+    )
+
+    assert transition.rate_conversion_type == rate_conversion_type
+    with patch(
+        "vivarium_public_health.disease.transition.rate_to_probability", return_value=1.0
+    ) as mock_rate_to_probability:
+        idx = pd.Index(list(range(10)))
+        transition._probability(idx)
+        calls = mock_rate_to_probability.call_args_list
+        for call_args in calls:
+            arg1, arg2 = call_args[0][0], call_args[1]["rate_conversion_type"]
+            assert arg1.index.equals(idx)
+            assert arg2 == rate_conversion_type
```

Comment:
Does this not work?
```python
mock_rate_to_probability.assert_called_once_with(idx, rate_conversion_type)
```

Reply:
No because they take different args but I should be able to get it to work like this

Reply:
Im going to see if there is a way to do that with a series

Reply:
Updated